README.md

```diff
@@ -45,6 +45,7 @@ from typing import List, Tuple
 MINIO_ENDPOINT = 'minio.your-company.co.uk'
 MINIO_EXTERNAL_ENDPOINT = "external-minio.your-company.co.uk"  # Default is same as MINIO_ENDPOINT
 MINIO_EXTERNAL_ENDPOINT_USE_HTTPS = True  # Default is same as MINIO_USE_HTTPS
+MINIO_REGION = 'us-east-1'  # Default is set to None
```

Comment:
i think that this comment should state that the default value now is `us-east-1`, not `None`